Ledger articles (#302)
moondance-labs/tanssi-docs: builders/toolkit/substrate-api/wallets/ledger.md

Ledger devices are hardware wallets designed to store the private keys offline. They are used to verify and sign the transactions, but they still need a software layer to provide the UI that interacts with the networks, builds the transactions, and sends the signed transactions back to the network once the user has verified them.

This guide takes you through all the necessary steps to use your Ledger device with Substrate Tanssi chains.

## Setting Up Your Ledger Device {: #setting-up-ledger-device }

If you have a brand new Ledger device, refer to the [official website](https://support.ledger.com/article/4404389503889-zd){target=\_blank} for a guide on getting it started with the initial setup.

Now, with your Ledger already initiated, install the _Polkadot_ app taking the following steps:

1. Open the Ledger Live App on your computer.
2. Go to My Ledger.
3. Connect and unlock the device.

Finally, search for Zondax’s Polkadot (DOT) app in Ledger Live and install it on your device.

!!! note

>     Why Polkadot? Tanssi is built using Substrate, the same framework that powers Polkadot. As a result, even though Tanssi and Polkadot are separate networks, they share the same cryptographic signature schema. This means that the Ledger app named Polkadot (DOT) is fully compatible with Tanssi.

Reviewer: ```suggestion
    Why Polkadot? Tanssi is built using Substrate, the same framework that powers Polkadot. As a result, even though Tanssi and Polkadot are separate networks, they share the same cryptographic signature schema. This means that the Ledger app named **Polkadot (DOT)** is fully compatible with Tanssi.
```

![Install Polkadot in Ledger Live](/images/builders/toolkit/substrate-api/wallets/ledger/ledger-1.webp)

And that's it. Your device now has a Substrate account and is able to sign transactions on Tanssi and any non-EVM Tanssi-powered appchain.

> ## Adding the Ledger to a Software Wallet {: #adding-Ledger--software-wallet }

Reviewer 2: Same comments as before

> As presented in the [introduction](#introduction), a Ledger hardware wallet provides secure, offline storage for private keys, allowing users to verify and sign transactions. However, by design, it can't interact with blockchains or dApps by itself, nor does it offer a UI for managing assets. To complement the device, a software wallet is required. The user can choose any Substrate wallet, such as [Talisman](/builders/toolkit/substrate-api/wallets/talisman/){target=\_blank} or [SubWallet](/builders/toolkit/substrate-api/wallets/subwallet/){target=\_blank}.

Reviewer 2: Hot wallet

> For demonstration purposes, we'll show how to configure Talisman to work with your hardware wallet. To follow along with the steps, make sure you have [Talisman installed](/builders/toolkit/substrate-api/wallets/talisman/#setting-up-talisman){target=\_blank} in your browser, then open the extension and:

Reviewer 2: ```suggestion
For demonstration purposes, we'll show how to configure Talisman to work with your hardware wallet, but these steps are generally applicable to any other Substrate-enabled wallet that supports Ledger. To follow along with the steps, make sure you have [Talisman installed](/builders/toolkit/substrate-api/wallets/talisman/#setting-up-talisman){target=\_blank} in your browser, then open the extension and:
```

1. Press the **+** icon (Add Account).
2. Click on the **Connect** tab.
3. Select **Connect Ledger**.

![Connect Ledger](/images/builders/toolkit/substrate-api/wallets/ledger/ledger-2.webp)

On the following screen, take the following steps:

1. Select **Polkadot**. Other options will be presented below.
2. Select **Polkadot** from the dropdown.
3. Select **Polkadot App** from the box. You'll be prompted to connect your ledger and open the Polkadot app.
4. Make sure that your ledger is successfully connected.
5. Click on **Continue**.

![Connect Ledger](/images/builders/toolkit/substrate-api/wallets/ledger/ledger-3.webp)

> In the next step, you will be presented with a list of derived accounts. Select the first one, click **Continue**, and that's all! Your Talisman wallet can now sign transactions using your ledger device.

Reviewer 2: ```suggestion
In the next step, you will be presented with a list of derived accounts. Select the one you want to import, click **Continue**, and that's all! Your Talisman wallet can now sign transactions using your ledger device.
```

> In the next step, you will be presented with a list of derived accounts. Select the one you want to import, click **Continue**, and that's all! Your Talisman wallet can now sign transactions using your ledger device.

Reviewer: ```suggestion
In the next step, you will be presented with a list of derived accounts. Select the one you want to import, click **Continue**, and that's all! Your Talisman wallet can now sign transactions using your Ledger device.
```
